spec/features/customer_details_spec.rb

```diff
@@ -0,0 +1,100 @@
+Capybara.app = Sinatra::Application
+
+describe 'customer details', type: :feature do
```

Comment:
I know I guided you to this 
Was thinking about the directory structure a bit more. I'm wondering whether `spec/ui/...` is a better directory structure to `spec/features/...`

Reasoning

* We don't want to test the "feature" - that should be covered by acceptance tests.
* We want to cover the UI code, and make sure it's wired up correctly. 
* We know what code is in the UI, and we know what tests are valuable to write
* We avoid writing tests for the business rules here.